Makefile

```diff
@@ -62,13 +62,13 @@ run_sp_mocks_backend_image:
 	docker run -p 8000:8000 -t sp_mocks_backend
 
 push_layer8_server_image:
-	aws lightsail push-container-image --region ca-central-1 --service-name aws-container-service-t1 --label layer8-server-version-18 --image layer8-server:latest
+	aws lightsail push-container-image --region ca-central-1 --service-name aws-container-service-t1 --label layer8-server-version-stable-1 --image layer8-server:latest
```

Comment:
suggestion: versioning is not necessarily for non production deployment, i suggest to use commit hash as label so you could determine up to which commit the code was deployed

Reply:
This idea is also good.